test-resources/odata-service-specs-e2e/v4/API_TEST_SRV/API-TEST_SRV.edmx

```diff
@@ -46,6 +47,62 @@
                 <Property Name="KeyToTestEntity" Type="Edm.Int32" Nullable="false"/>
                 <Property Name="StringProperty" Type="Edm.String" MaxLength="111"/>
             </EntityType>
+            <EntityType Name="TestEntity50Prop">
```

Comment:
perhaps name the spec file `50-properties-memeory-test-service` or so to make clear where or for what it is used. Also feel free to add a comment in the service above the entity to explain why so many entites

Reply:
I extended the spec for the cap e2e tests by adding a new entity.
I can add a comment.